ext/AvizExt/viz/coral_dynamics.jl

```diff
@@ -0,0 +1,275 @@
+    series_opts[:labels] = get(series_opts, :labels, taxa_names)
+
+    # Get default axis options
+    xtick_vals = get(axis_opts, :xticks, _time_labels(timesteps(relative_taxa_cover)))
+    xtick_rot = get(axis_opts, :xticklabelrotation, 2 / π)
+    for (idx, scen_name) in enumerate(keys(scen_groups))
+        ax = Axis(g[idx, 1]; title=String(scen_name), xticks=xtick_vals, xticklabelrotation=xtick_rot, axis_opts...)
+
+        taxonomy_series_confint!(
+            ax,
+            relative_taxa_cover[scenarios=scen_groups[scen_name]],
+            colors;
+            show_confints=show_confints,
+            series_opts=series_opts
+        )
+
+    end
+    return nothing
+end
+function taxonomy_series_confint!(
+    ax::Axis,
+    relative_taxa_cover::YAXArray,
+    colors::Union{Vector{Symbol},Array{RGBA{T},1}};
+    show_confints=true,
+    show_legend=true,
+    series_opts::Dict{Symbol,<:Any}=Dict{Symbol,Any}()
+)::Nothing where {T<:Float32}
+    n_timesteps::Int64 = length(relative_taxa_cover.timesteps)
+    n_groups::Int64 = length(relative_taxa_cover.taxa)
+
+    # Plot and calculate confidence intervals
+    confints = zeros(n_timesteps, n_groups, 3)
+    for (idx, taxa) in enumerate(relative_taxa_cover.taxa)
+        confints[:, idx, :] = series_confint(relative_taxa_cover[taxa=At(taxa)])
+        show_confints ? band!(
+            ax, 1:n_timesteps, confints[:, idx, 1], confints[:, idx, 3];
+            color=(colors[idx], 0.4)
+        ) : nothing
+    end
+
+    # Plot series
+    series!(ax, 1:n_timesteps, confints[:, :, 2]'; solid_color=colors, series_opts...)
+    show_legend ? axislegend(ax) : nothing
+
+    return nothing
+end
+
+"""
+    taxonomy_intervened_series_confint!( g::Union{GridLayout, GridPosition}, relative_taxa_cover::YAXArray, scen_groups::Dict{Symbol,BitVector}, colors::Union{Vector{Symbol},Vector{RGBA{T}}}; show_confints::Bool=true, axis_opts::Dict{Symbol,<:Any}=Dict{Symbol,Any}(), series_opts::Dict{Symbol,<:Any}=Dict{Symbol,Any}())::Nothing where {T<:Float32}
+    taxonomy_intervened_series_confint!( ax::Axis, relative_taxa_cover::YAXArray, colors::Union{Vector{Symbol},Vector{RGBA{T}}}, scen_groups::Dict{Symbol,BitVector}; show_confints::Bool=true, show_legend::Bool=true, series_opts::Dict{Symbol,<:Any}=Dict{Symbol,Any}())::Nothing where {T<:Float32}
+
+Plot relative cover, comparing taxa cover between different scenario groups. Each plot is a
+different coral taxonomy or functional group.
+"""
+function taxonomy_intervened_series_confint!(
+    g::Union{GridLayout, GridPosition},
+    relative_taxa_cover::YAXArray,
+    scen_groups::Dict{Symbol,BitVector},
+    colors::Union{Vector{Symbol},Vector{RGBA{T}}};
+    show_confints::Bool=true,
+    axis_opts::Dict{Symbol,<:Any}=Dict{Symbol,Any}(),
+    series_opts::Dict{Symbol,<:Any}=Dict{Symbol,Any}()
+)::Nothing where {T<:Float32}
+    taxa_names = human_readable_name(String.(functional_group_names()), title_case=true)
```

Comment:
I think this function deserves another method accepting a `Vector{Symbol}` so you don't have to do this conversion when calling the function (that could help other including your future self)